pkg/manifest/manifest.go

```diff
@@ -38,10 +39,10 @@ type OAuth struct {
 	TokenEndpoint *URLDefinition
 }
 
-// GetTokenEndpointValue returns the defined token endpoint or an empty string if it's not set.
+// GetTokenEndpointValue returns the defined token endpoint or the default token endpoint if none is defined
 func (o OAuth) GetTokenEndpointValue() string {
-	if o.TokenEndpoint == nil {
-		return ""
+	if o.TokenEndpoint == nil || o.TokenEndpoint.Value == "" {
```

Comment:
nit: for this to be an empty string, a user would have to explicitly set it to "" or an env var that resolves to "" right? 
I wonder if we shouldn't print a warning for that case, before defaulting to the default URL. If you explicitly set this to empty, you probably made a mistake trying to set it to an actual value 🤔

Reply:
it would be beneficial, however, it would be nicer to have that warning once and not whenever that method is called.